src/reducers/countriesReducer.js

```diff
@@ -9,6 +9,8 @@ const countriesReducer = (state = {}, action) => {
     case RECEIVE_CURRENT_USER:
       if (action.payload.countries) {
         return action.payload.countries;
+      } else {
+        return merge({}, state);
```

Comment:
When would we have receive_current_user without countries?

Reply:
We are receiving current user in two different situations - 1) we just refreshed the app but somebody is logged in, in which case we are getting all the countries along with the current user and 2) we just logged in, but we don't need the countries because we already got them when the app first loaded.